circe-area.js

```diff
@@ -0,0 +1,4 @@
+const circleArea = (radius) => {
+  return `${radius ** 2 * Math.PI}`;
```

Comment:
Here you are unnecessarily converting a number to a string. You could just return the number instead:
```suggestion
  return radius ** 2 * Math.PI;
```